src/index/coinstatsindex.cpp

```diff
@@ -155,22 +172,22 @@ bool CoinStatsIndex::CustomAppend(const interfaces::BlockInfo& block)
 
                 // Skip unspendable coins
                 if (coin.out.scriptPubKey.IsUnspendable()) {
-                    read_out.second.total_unspendable_amount += coin.out.nValue;
-                    read_out.second.total_unspendables_scripts += coin.out.nValue;
+                    block_unspendable += coin.out.nValue;
+                    new_value.second.block_unspendables_scripts += coin.out.nValue;
```

Comment:
Isn't incrementing `new_value.second.block_unspendables_scripts` undefined behavior since it never gets initialized? (and the other `block_*` members of DBVal)

Reply:
> Adding {0} default initializers in the struct DBVal declaration seems sensible?

Done